bittensor_cli/src/commands/root.py

```diff
@@ -719,58 +719,72 @@ async def _get_list() -> tuple:
         )
         return sm, rn, di, ts
 
-    table = Table(
-        Column(
-            "[bold white]UID",
-            style="dark_orange",
-            no_wrap=True,
-        ),
-        Column(
-            "[bold white]NAME",
-            style="bright_cyan",
-            no_wrap=True,
-        ),
-        Column(
-            "[bold white]ADDRESS",
-            style="bright_magenta",
-            no_wrap=True,
-        ),
-        Column(
-            "[bold white]STAKE(\u03c4)",
-            justify="right",
-            style="light_goldenrod2",
-            no_wrap=True,
-        ),
-        Column(
-            "[bold white]SENATOR",
-            style="dark_sea_green",
-            no_wrap=True,
-        ),
-        title=f"[underline dark_orange]Root Network[/underline dark_orange]\n[dark_orange]Network {subtensor.network}",
-        show_footer=True,
-        show_edge=False,
-        expand=False,
-        border_style="bright_black",
-    )
+
     with console.status(
         f":satellite: Syncing with chain: [white]{subtensor}[/white] ...",
         spinner="aesthetic",
     ):
+            
         senate_members, root_neurons, delegate_info, total_stakes = await _get_list()
+        total_tao = sum(float(Balance.from_rao(total_stakes[neuron.hotkey])) for neuron in root_neurons)
 
-    if not root_neurons:
-        err_console.print(
-            f"[red]Error: No neurons detected on network:[/red] [white]{subtensor}"
+        table = Table(
+            Column(
+                "[bold white]UID",
+                style="dark_orange",
+                no_wrap=True,
+                footer=f"[bold]{len(root_neurons)}[/bold]"
+            ),
+            Column(
+                "[bold white]NAME",
+                style="bright_cyan",
+                no_wrap=True,
+            ),
+            Column(
+                "[bold white]ADDRESS",
+                style="bright_magenta",
+                no_wrap=True,
+            ),
+            Column(
+                "[bold white]STAKE(\u03c4)",
+                justify="right",
+                style="light_goldenrod2",
+                no_wrap=True,
+                footer=f"{total_tao:.2f} (\u03c4) "
+            ),
+            Column(
+                "[bold white]SENATOR",
+                style="dark_sea_green",
+                no_wrap=True,
+            ),
+            title=f"[underline dark_orange]Root Network[/underline dark_orange]\n[dark_orange]Network {subtensor.network}",
```

Comment:
Just curious, is there a missing [/dark_orange] for the "Network"?

Reply:
If the line is ending, we dont have to necessarily add a closing tag for colour